src/olympia/reviewers/tests/test_views.py

```diff
@@ -3578,8 +3576,8 @@ def test_item_history_pagination(self):
             version_factory(
                 addon=addon, version=f'1.{i}', created=self.days_ago(365 - i)
             )
-        with self.assertNumQueries(59):
-            # FIXME: 59 is obviously still too high, but it's a starting point.
+        with self.assertNumQueries(62):
+            # FIXME: 62 is obviously still too high, but it's a starting point.
```

Comment:
The queries were always there potentially. This is caused by the addition of the `CHANGE_STATUS` to the list of actions considered in review page, and because this test does cause such an activity to be recorded and therefore shown. It doesn't scale up though (it's one for every user+add-on+translations, the comments reflect that - I unfortunately had to go through them all because they weren't in the right order anymore because of the add-ons categories change a while back for some reason)